namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-data-compliance-dev/05-serviceaccount-circleci.yaml

```diff
@@ -4,38 +4,17 @@ metadata:
   name: circleci
   namespace: prison-data-compliance-dev
 
----
-kind: Role
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: circleci
-  namespace: prison-data-compliance-dev
-rules:
-  - apiGroups:
-      - ""
-    resources:
-      - "pods/portforward"
-    verbs:
-      - "create"
-  - apiGroups:
-      - ""
-    resources:
-      - "pods"
-    verbs:
-      - "get"
-      - "list"
-
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: circleci
-  namespace: prison-data-compliance-dev
+  namespace: prison-data-compliance-dev 
```

Comment:
Is the whitespace here an issue?

Reply:
I dont think so, but I've removed it anyway